lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+class Menu
+
+  attr_reader :dishes_available
+
+  def initialize 
+    @dishes_available = { "Duck Pancakes" => 4.99,
+      "Chicken Chow Mein" => 3.85,
```

Comment:
So I might be tempted to set this hash to a constant (such as `CURRENT_MENU`), then initialize `@dishes_available` to that constant. Why? Because menu is data, and we should always be trying to separate data from behaviour. 

A `CURRENT_MENU` constant could be linked to an external file for example, that a non-technical user updates when prices change etc. This facility would allow editing of the data without interfering with the logic in our program. In this case it's such a simple class that the benefits of this aren't immediately obvious, but as our program grows in complexity so does our need for separation of concerns.